app/views/site/genome/_browse_igv_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if study_file.generation %>
-<%= link_to 'Browse in genome', '#study-visualize', data: {filename: ERB::Util.url_encode(study_file.upload_file_name)}, class: 'bam-browse-genome btn btn-primary dl-link other-file' %>
+<%= link_to 'Browse in genome', '#', data: {filename: ERB::Util.url_encode(study_file.upload_file_name)}, class: 'bam-browse-genome btn btn-primary dl-link other-file' %>
```

Comment:
why shouldn't that be a link to the visualization tab?  Even if we're actually handling the navigation in javascript, it seems user-friendlier to let them know where going to take them to a different tab location

Reply:
Nice point -- restored in 5022640.